app/views/mortgage_calculator/land_and_buildings_transaction_taxes/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for(:head) do %>
+  <script>
+   window.calculator_config = <%= raw calculator_config_json %>;
+  </script>
+<% end %>
+
+<div ng-controller="CalculatorCtrl" class="mortgagecalc__container stamp-duty">
+  <% @lbtt.errors.full_messages.each do |message| %>
+    <span style="color:red;"><%= message %></span>
```

Comment:
Are we sure we want to leave this inline HTML style in here?

```html
<span style="color:red;"><%= message %></span>
```

Reply:
This is the styling thats used in the current stamp duty calculator do you feel strongly we should update this?

Reply:
@davidtrussler @johnplayer1982 what do you think please?

Reply:
It's a bit odd that that's in the current one and we're not even using a colour from our palette! My feeling is to come back to this separately and change them all together to ensure consistency.

Reply:
Thanks @davidtrussler. @gotbadger that clarifies matters further. We can leave this as is as it will be part of a more extensive work at a later date